cypress/integration/saving.spec.ts

```diff
@@ -1,26 +1,17 @@
 describe('Saving', () => {
   it('Should allow you to save a file for the first time', () => {
     cy.setMockAuthToken();
-
-    const sourceFileToBeCreated = {
-      id: 'source-file-id',
-      name: 'My awesome source file',
-      owner: {
-        id: 'id',
-      },
-    };
-
-    cy.intercept('source-file-id.json?sourceFileId=source-file-id', {
-      pageProps: {
-        id: sourceFileToBeCreated.id,
-        data: sourceFileToBeCreated,
-      },
-    });
     cy.interceptGraphQL({
```

Comment:
What are the reasons for the changes in this file?

Reply:
Not sure why these are removed. Could've been a mistake when we were trying to fix the cypress tests.